src/gsy_e/models/leaves.py

```diff
@@ -244,16 +236,14 @@ class ForecastSCMPV(CoefficientLeaf):
     "LoadHours": SCMLoadHours,
     "LoadProfile": SCMLoadProfile,
     "Storage": SCMStorage,
-    "PV": SCMPV,
-    "PredefinedPV": SCMPredefinedPV,
```

Comment:
What if the gsy-web sends a `PredefinedPV` ? Or will this be resolved by https://github.com/gridsingularity/gsy-web/pull/1966 ?

Reply:
Exactly, this will be resolved by the other PR. This should not happen with GSy Maps and SCM though, no Predefined PVs are configured.